corehq/apps/accounting/tasks.py

```diff
@@ -227,6 +235,19 @@ def send_subscription_reminder_emails(num_days, exclude_trials=True):
             subscription.send_ending_reminder_email()
 
 
+def send_subscription_reminder_emails_dimagi_contact(num_days):
+    today = datetime.date.today()
+    date_in_n_days = today + datetime.timedelta(days=num_days)
+    ending_subscriptions = (Subscription.objects
+                            .filter(is_active=True)
+                            .filter(date_end=date_in_n_days)
```

Comment:
I'm not totally sure about this. I was under the impression that every task we set goes into a queue in redis, and Celery grabs tasks from the queue whenever it is free to do so. If celery goes down, the queue still gets populated, and celery catches up when it comes back online. I might have used the wrong decorator for this functionality though...